mri/operators/fourier/non_cartesian.py

```diff
@@ -34,6 +34,14 @@
                   "the master release. Till then you cannot use NUFFT on GPU")
     pass
 
+try:
+    from gpuNUFFT import NUFFTOp
+    gpunufft_available = True
```

Comment:
a neater way to do it is to have that in the `else` of the `try/except`. You generally want a single statement in the `try` to make sure you control the error source

Reply:
There is else for try except?? :o

Reply:
yes the extended `try/except` statement is:
```python
try:
  one-liner statement
except (Error1, Error2):
  throw custom warning
else:
  in case you want to do something only when try has succeeded
finally:
  general clean-up, for example if you opened a file or what-not
```

Reply:
Hmmm

Reply:
So I can skip finally? Cause we dont exactly have any cleanup right?

Reply:
yes yes you don't need `finally` in this case!